app/models/status.rb

```diff
@@ -10,7 +10,7 @@ class Status < ApplicationRecord
 
   belongs_to :application, class_name: 'Doorkeeper::Application'
 
-  belongs_to :account, inverse_of: :statuses, counter_cache: true
+  belongs_to :account, inverse_of: :statuses, counter_cache: true, touch: true
```

Comment:
We wanna have an update timestamp in the Atom feed header, but we don't wanna additionally fetch which entry is the latest when rendering the feed (e.g. when paginating through the feed, wouldn't be able to use the loaded set, needs extra query). With the touch, `account.updated_at` will be up-to-date with latest status.